src/engraving/style/style.h

```diff
@@ -97,6 +97,9 @@ class MStyle
 
     std::array<PropertyValue, size_t(Sid::STYLES)> m_values;
     std::array<Millimetre, size_t(Sid::STYLES)> m_precomputedValues;
+
+    void readVersion(String versionTag);
+    int _version;
```

Comment:
1. Please rename it to m_version
2. Assign a default value (0)

Reply:
done!